ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -143,20 +142,17 @@ def _map_sample(self, sample, exclude_fields):
             rendered = jinja2.Template(
                 self.template, undefined=jinja2.StrictUndefined
             ).render(**filtered_sample, json_dump=json.dumps, text_proc=text_proc)
-
+        except text_proc.FormatASRError as e:
+            print(f"Format ASR Error {e}")
+            sample[self.new_column_name] = ""
```

Comment:
Setting this to "" instead of None because of hf dataset type errors associated with setting the value to None.

Reply:
Hmm, so now it's not allowed to set the column to "" which is an artificial constraint, but I don't see a hugely better solution. Using a pre-specified string might be better, but not ideal.

Please mention that "" is not allowed somewhere though.

Reply:
All samples with `self.new_column_name = None` will be filtered, right? If yes, maybe add a comment here to make this point explicit?